.goreleaser.yaml

```diff
@@ -7,13 +7,30 @@ before:
 builds:
   - env:
       - CGO_ENABLED=0
+    binary: vet
     goos:
       - linux
       - darwin
     goarch:
       - amd64
       - arm64
 
+release:
+  # for prerelease it doesn't build and distribute
+  prerelease: auto
+
+universal_binaries:
+  - replace: true
+
+brews:
+  - 
+    name: vet
+    homepage: https://safedep.io
+    description: "SafeDep vet is a tool for identifying open source software supply chain risks"
+    tap:
+      owner: safedep
+      name: homebrew-vet
```

Comment:
We need to create this repository and configure this job with a PAT that has access to push to the repo.

Also is it possible to create a common brew repo, something like `safedep/homebrew` and push `vet` formula there along with any other tool that we develop in future?

Reply:
Fixed this to be `homebrew-tap` as the releaser looks for `homebrew-*`. So now we can leverage this for all the SafeDep tooling 👍